data/recipes/upload_web_ts.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "upload_web_ts",
+    "description": "Uploads a parsed web history or EVTX records stored in a CSV or Plaso file to Timesketch.",
```

Comment:
Given that this is very similar to `upload_ts`, I'd try to stick to the other recipe's description and add "and uses web artifact analyzers" or something.

Reply:
reworded